cdc/log.cc

```diff
@@ -746,6 +746,28 @@ class transformer final {
             };
 
             int batch_no = 0;
+            if (!p.static_row().empty()) {
```

Comment:
Do I understand correctly that this code assumes static row will be split into a separate mutation even when it's timestamp and ttl are the same as in some other mutations of clustering rows?

Is current splitting working this way?

Reply:
Fortunately, yes. Static rows are always split, regardless of the timestamps (as evidenced [here](https://github.com/scylladb/scylla/blob/325c3e13ebf1e4b295579b7a9c0dddc01a4e4ec0/cdc/split.cc#L287)). I will extend the preimage test for the static row to handle mixed mutations (with static and clustering rows) as well.

Reply:
Great! Keep moving then - there's nothing to see here :D